sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -504,16 +504,15 @@ namespace Namespace
 {
   public class DeclarationStatement
   {
-    public int Main(Dictionary<string, string> dictionary, string key)
+    public int Main(Dictionary<string, int> dictionary, string key)
```

Comment:
do we still have a UT for reference type declarations? because I see here only value-type ones

Reply:
No, indeed. I've added a new test to assert that we don't add the **NotNull** constraint for reference types.